### ISSUE TITLE
Reason for removed git docs

### ISSUE THREAD

**Issue by F0Xde** [tomsquest/docker-radicale#96]
Was just wondering whether there is any reason for [doc: remove part about integrating Radicale and Git ](https://github.com/tomsquest/docker-radicale/commit/c82e2bb41daa8dc2eef1d87ded26f006aaaa0898) that one should be aware of, or if using git versioning is still fine to do?

**Comment by shtingeri** [comment]
TLDR;  change git hook command to
`hook = ([ -d .git ] || git init && git config user.name user && git config user.email user@example.com && echo -e ".Radicale.cache\n.Radicale.lock\n.Radicale.tmp-*" > .gitignore) && git add -A && (git diff --cached --quiet || git commit -m "Changes by "%(user)s)`


Neither the git hook command from official docs nor this repo's worked for me.
And there's little info to be known from the log other than error status code.
Please refer to  [https://github.com/Kozea/Radicale/issues/1181#issuecomment-914209910](url) for possible resolution. At least It worked for me.

**Comment by tomsquest** [comment]
@F0Xde 
- Git versioning is handled by Radicale itsefl, so it should work. The idea is to init the git repo before, so in the data volume. Or as  @shtingeri pointed, the hook can do everything if the directory is not a git repo.

And btw, I remove the documentation, because I don't use the hook feature of radicale so I cannot make sure it work correctly now and in the furure. So basically it boils down to reducing the maintenance.

**Comment by F0Xde** [comment]
Thank you very much for the explanation. I went the simple route and initialized the git repo manually beforehand, so for everything seems to work great.
